installers/osie/ipxe_script_test.go

```diff
@@ -54,7 +54,7 @@ func TestScript(t *testing.T) {
 			s := ipxe.Script{}
 			s.Echo("Packet.net Baremetal - iPXE boot")
 			s.Set("iface", "eth0").Or("shell")
-			s.Set("tinkerbell", "http://127.0.0.1")
+			s.Set("boots", "http://127.0.0.1")
```

Comment:
I think we can actually get rid of this, since osie doesn't use it, but can leave that for another pr

Reply:
Shall it be changed back to `tinkerbell`?